syntax = "proto3";

service SimpleApp {
  rpc LoginUser(LoginUserRequest) returns (LoginUserResponse) {}
  rpc RegisterUser(RegisterUserRequest) returns (RegisterUserResponse) {}
  rpc GetInfo(GetInfoRequest) returns (GetInfoResponse) {}
}
                                                                                                                                                                                                                                  
message LoginUserRequest {
  string username = 1;
  string password = 2;
}

message LoginUserResponse {
  string message = 1;
}

message RegisterUserRequest {
  string username = 1;
  string password = 2;
}

message RegisterUserResponse {
  string message = 1;
}

message GetInfoRequest {
  string id = 1;
}

message GetInfoResponse {
  string message = 1;
}